### ISSUE TITLE
No output when running action

### ISSUE THREAD

**Issue by berniedurfee-renaissance** [reload/github-security-jira#346]
I don't see any output when running the action.

```
sync-dependabot-alerts:
        runs-on: ubuntu-latest
        steps:
            - name: "Sync Dependabot security alerts to Jira issues"
              uses: reload/github-security-jira@v1.5.0
              env:
                GH_SECURITY_TOKEN: ${{ secrets.GitHubSecurityToken }}
                JIRA_TOKEN: ${{ secrets.JiraApiToken }}
                JIRA_HOST: https://xyz.atlassian.net/
                JIRA_USER: me@company.com
                JIRA_PROJECT: ABC
                JIRA_ISSUE_TYPE: Bug
                JIRA_WATCHERS: me@company.com
                JIRA_ISSUE_LABELS: Dependabot
```

Output:

```
2023-12-18T16:54:13.4088137Z ##[group]Run reload/github-security-jira@v1.x
2023-12-18T16:54:13.4088657Z env:
2023-12-18T16:54:13.4089366Z   ...
2023-12-18T16:54:13.4093714Z ##[endgroup]
2023-12-18T16:54:13.4391367Z ##[command]/usr/bin/docker run ...
2023-12-18T16:54:16.2292260Z Cleaning up orphan processes
```

I expect to see some output like:

```
No alerts found.
```

**Comment by arnested** [comment]
Agreed, we could definitely use a more verbose output in the log.

**Comment by arnested** [comment]
Fixed in #347.
